spark/v3.4/spark/src/main/java/org/apache/iceberg/spark/SparkSQLProperties.java

```diff
@@ -27,6 +27,10 @@ private SparkSQLProperties() {}
   // Controls whether vectorized reads are enabled
   public static final String VECTORIZATION_ENABLED = "spark.sql.iceberg.vectorization.enabled";
 
+  // Controls which Parquet reader to use for vectorization
+  public static final String PARQUET_READER_TYPE = "spark.sql.iceberg.parquet.reader-type";
+  public static final ParquetReaderType PARQUET_READER_TYPE_DEFAULT = ParquetReaderType.COMET;
```

Comment:
The default should be the existing Iceberg reader. We can only consider the migration after some time and only if the community agrees to it.

Reply:
Right. I am only setting Comet to default for testing purpose and want to make sure all tests pass with Comet. Will set back to iceberg reader.